cli/src/test/java/com/devonfw/tools/ide/tool/tomcat/TomcatTest.java

```diff
@@ -37,9 +38,9 @@ public void testTomcat() {
     Path javaTomcatPath = context.getToolRepositoryPath().resolve("default").resolve("java").resolve("java").resolve(javaVersionTomcat);
 
     // act
-    javaCommandlet.arguments.addValue("--version");
-    javaCommandlet.run();
-    tomcatCommandlet.run();
+    // hack to capture output of test script. Original processMode was BACKGROUND.
```

Comment:
For the record: IMHO you wrote this before you re-introduced the fallback to DEFAULT_CAPTURE for all tests. This leftover message is now slightly incorrect and could be confusing.

Reply:
I've reverted these hacks now.